test/unit/adapters/commms-gatekeepr.spec.ts

```diff
@@ -207,3 +216,72 @@ describe('when getting the private voice chat credentials', () => {
     })
   })
 })
+
+describe('when ending a private voice chat', () => {
+  let callId: string
+  let address: string
+
+  beforeEach(() => {
+    callId = 'test-call-id-123'
+    address = '0xBceaD48696C30eBfF0725D842116D334aAd585C1'
+  })
+
+  describe('and the request resolves with a 200 status code', () => {
+    beforeEach(() => {
+      fetchMock.mockResolvedValueOnce({
+        ok: true,
+        status: 200
+      })
+    })
+
+    it('should resolve successfully', async () => {
+      await expect(commsGatekeeper.endPrivateVoiceChat(callId, address)).resolves.toBeUndefined()
+    })
+
+    it('should make the correct API call', async () => {
```

Comment:
Should we use a more specific description like "should make a delete request to [...]" or similar?